editor/game_data.cpp

```diff
@@ -213,6 +214,10 @@ namespace NEONnoir
         auto savefile = std::ifstream{ filename };
         if (savefile)
         {
+            // Vairn, get the path of the script, and set it as the working directory, so we no longer need to use full paths.
+            std::string filepath = GetPathFromFilename(filename);
```

Comment:
Since I'm already using std::filesystem we can do something like:
```c++
            auto data_path = fs::path{ filename }.parent_path();
            fs::current_path(data_path);
```

Reply:
interesting, when I tried to use fs, I had issues.
if it works for go, go for it. :)